core/src/main/java/org/stellar/anchor/util/SepHelper.java

```diff
@@ -118,14 +118,17 @@ public static BigDecimal validateAmount(String messagePrefix, String amount, boo
 
   public static void validateAmountLimit(String messagePrefix, String amount, Long min, Long max)
       throws AnchorException {
-    BigDecimal sAmount = validateAmount("", amount);
-    BigDecimal bdMin = new BigDecimal(min);
-    BigDecimal bdMax = new BigDecimal(max);
-    if (sAmount.compareTo(bdMin) < 0) {
-      throw new BadRequestException(String.format("%samount less than min limit", messagePrefix));
-    }
-    if (sAmount.compareTo(bdMax) > 0) {
-      throw new BadRequestException(String.format("%samount exceeds max limit", messagePrefix));
+    if (min != null && max != null) {
```

Comment:
I think we may start adding comments for the logics that can be confusing. 
```suggestion
    // since min and max are optional in assets, compare only if defined.
    if (min != null && max != null) {
```

Reply:
Same here